docs-source/content/samples/simple/azure-kubernetes-service/troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+Warning  FailedMount  119s                 kubelet, aks-nodepool1-58449474-vmss000001  Unable to mount volumes for pod "domain1-create-weblogic-sample-domain-job-qqv6k_default(15706980-73cb-11ea-b804-b2c91b494b00)": timeout expired waiting for volumes to attach or mount for pod "default"/"domain1-create-weblogic-sample-domain-job-qqv6k". list of unmounted volumes=[weblogic-sample-domain-storage-volume]. list of unattached volumes=[create-weblogic-sample-domain-job-cm-volume weblogic-sample-domain-storage-volume weblogic-credentials-volume default-token-zr7bq]
+Warning  FailedMount  114s (x9 over 4m2s)  kubelet, aks-nodepool1-58449474-vmss000001  MountVolume.SetUp failed for volume "wls-azurefile" : Couldn't get secret default/azure-secrea
+```
+
+#### Fail to access Administration Console
+
+Here are some common reasons for this failure, along with some tips to help you investigate.
+
+* **Create WebLogic domain job fails**
+
+Check the deploy log and find the failure details with `kubectl describe pod podname`.
+Please go to 1. Getting pod error details.
+
+* **Process of starting the servers is still running**
+
+Check with `kubectl get svc` and if domainUID-admin-server, domainUID-managed-server1 and domainUID-managed-server2 are not listed,
+we need to wait some more for the Administration Server to start.
+
+The following output is an example of when the Administration Server has started.
+
+```bash
+$ kubectl get svc
+NAME                               TYPE           CLUSTER-IP    EXTERNAL-IP     PORT(S)              AGE
+domain1-admin-server               ClusterIP      None          <none>          30012/TCP,7001/TCP   7m3s
+domain1-admin-server-ext           NodePort       10.0.78.211   <none>          7001:30701/TCP       7m3s
+domain1-admin-server-external-lb   LoadBalancer   10.0.6.144    40.71.233.81    7001:32758/TCP       7m32s
+domain1-cluster-1-lb               LoadBalancer   10.0.29.231   52.142.39.152   8001:31022/TCP       7m30s
+domain1-cluster-cluster-1          ClusterIP      10.0.80.134   <none>          8001/TCP             1s
+domain1-managed-server1            ClusterIP      None          <none>          8001/TCP             1s
+domain1-managed-server2            ClusterIP      None          <none>          8001/TCP             1s
+internal-weblogic-operator-svc     ClusterIP      10.0.1.23     <none>          8082/TCP             9m59s
+kubernetes                         ClusterIP      10.0.0.1      <none>          443/TCP              16m
+```
+
+If services are up but the WLS Administration Console is still not available, use `kubectl describe domain` to check domain status.
+
+```bash
+$ kubectl describe domain domain1
+```
+
+Make sure the status of cluster-1 is `ServersReady` and `Available`. The status of admin-server, managed-server1 and managed-server2 should be `RUNNING`. Otherwise, the cluster is likely still in the process of becoming fully ready.
```

Comment:
admin-server, managed-server1 and managed-server2 ->` admin-server`, `managed-server1`, and `managed-server2` (code font)

Reply:
Will do.